wandb/sdk/wandb_login.py

```diff
@@ -272,6 +264,22 @@ def prompt_api_key(self) -> None:
 
         self.update_session(key, status=status)
         self._key = key
+        return key
+
+
+def _verify_login(key: str) -> bool:
```

Comment:
```suggestion
def _verify_login(key: str) -> None:
```

This method can't return False, and its return value is never checked.